src/test/java/TestMain.java

```diff
@@ -0,0 +1,52 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestMain {
+
+    @Test
+    public void shouldReturnTwentyThree() {
+        //when
```

Comment:
W każdym teście powinieneś mieć 3 sekcje - `given` - gdzie przygotowujesz sobie parametry wejściowe do testu, `when` - gdzie wywołujesz metodę którą chcesz testować, oraz `then` - gdzie upewniasz się że to co dostałeś spełnia Twoje oczekiwania. Ty wymieszałeś wszystkie te sekcje razem.

Reply:
Zobacz w source codzie z zajęć jak to powinno wyglądać.

Reply:
Wszystkie testy niżej też muszą zostać poprawione.